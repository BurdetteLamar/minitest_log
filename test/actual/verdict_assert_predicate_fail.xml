<log>
  <summary_ verdicts='3' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_assert_predicate?' type='fail'>
    <section_ name='Args=[&quot;x&quot;, :empty?, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_assert_predicate?' outcome='failed' id='0' message='Message 0'>
        <object_ class='String' value='&quot;x&quot;'/>
        <operator_ class='Symbol' value=':empty?'/>
        <exception_ class='Minitest::Assertion' message='Expected # encoding: UTF-8'>
          <backtrace_>
          </backtrace_>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[false, :nil?, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_assert_predicate?' outcome='failed' id='1' message='Message 1'>
        <object_ class='FalseClass' value='false'/>
        <operator_ class='Symbol' value=':nil?'/>
        <exception_ class='Minitest::Assertion' message='Expected false to be nil?.'>
          <backtrace_>
          </backtrace_>
        </exception_>
      </verdict_>
    </section_>
  </section_>
  <verdict_ method='verdict_assert_equal?' outcome='passed' id='error_count'>
    <expected_ class='Integer' value='0'/>
    <actual_ class='Integer' value='0'/>
  </verdict_>
</log>
