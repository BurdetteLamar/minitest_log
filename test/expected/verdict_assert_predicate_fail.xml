<log>
  <summary verdicts='3' failures='2' errors='0'/>
  <section name='Test' method='verdict_assert_predicate?' type='fail'>
    <section name='Args=[&quot;x&quot;, :empty?]'>
      <verdict method='verdict_assert_predicate?' outcome='failed' id='0'>
        <object>
          x
        </object>
        <operator>
          :empty?
        </operator>
        <exception class='Minitest::Assertion'>
          <message>
            Expected # encoding: UTF-8 &quot;x&quot; to be empty?.
          </message>
          <backtrace/>
        </exception>
      </verdict>
    </section>
    <section name='Args=[false, :nil?]'>
      <verdict method='verdict_assert_predicate?' outcome='failed' id='1'>
        <object>
          false
        </object>
        <operator>
          :nil?
        </operator>
        <exception class='Minitest::Assertion'>
          <message>
            Expected false to be nil?.
          </message>
          <backtrace/>
        </exception>
      </verdict>
    </section>
  </section>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <exp_value>
      0
    </exp_value>
    <act_value>
      0
    </act_value>
  </verdict>
</log>
